jazzmin/templatetags/jazzmin.py

```diff
@@ -178,15 +178,17 @@ def get_user_avatar(user: AbstractUser) -> str:
 
     # If we find the property directly on the user model (imagefield or URLfield)
     avatar_field = getattr(user, avatar_field_name, None)
-    if avatar_field:
-        if type(avatar_field) == str:
+    if avatar_field is not None:
+        if not avatar_field:
+            return no_avatar
```

Comment:
Optional - `no_avatar` is basically a constant so potentially could be moved outside the function to the top of the file (or to the settings).